src/test/java/com/thealgorithms/backtracking/FloodFillTest.java

```diff
@@ -0,0 +1,78 @@
+package com.thealgorithms.backtracking;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+
+class FloodFillTest {
+
+    @Test
+    void testForEmptyImage()
+    {
+        int image[][] = {};
+        int expected[][] = {};
+
+        int resultImage[][] = FloodFill.getFloodFillImage(image, 4, 5, 3, 2);
+        assertArrayEquals(expected, resultImage);
```

Comment:
```suggestion
        FloodFill.floodFill(image, 4, 5, 3, 2);
        assertArrayEquals(expected, image);
```

Reply:
oh okay, yeah I missed out on this. It could be done this way. Thanks @siriak